### ISSUE TITLE
invalid scop

### ISSUE THREAD

**Issue by chetanxhedamba** [paypal/paypalcheckout-ios#94]
Hello i am getting error error as per screen shot please kindly help me to solve this
![Simulator Screen Shot - iPhone 8 Plus - 2022-02-01 at 15 35 41](https://user-images.githubusercontent.com/91871479/152163701-2cbc13bc-a667-43a0-a92d-9e72e70dbc03.png)
 
- SDK version (`Checkout.version`): - 0.8400

**Comment by minhthenguyen** [comment]
Hi @chetanxhedamba 
Thank you for reporting the issue.
Can you please go to https://developer.paypal.com/developer/applications and select your application,
then under `SANDBOX APP SETTINGS`, enable `Log in with PayPal`, and in `Advanced Settings`, enable `Full name` and `Email`?

Please let us know if the issue persists after following above steps.

<img width="998" alt="Screen Shot 2022-02-03 at 10 32 42 PM" src="https://user-images.githubusercontent.com/80476442/152374381-84f71022-5801-47af-a7d9-f2c2498359e4.png">

**Comment by jonathajones** [comment]
Should be resolved with Minh's response above.

**Comment by rohitramkumar** [comment]
I'm having the same issue even after enabling the two checkboxes from the previous comment.

**Comment by rohitramkumar** [comment]
@jonathajones Anything you need from me to help debug this?

I have order IDs from the sandbox environment that were created during testing of this.

**Comment by jonathajones** [comment]
If you could post your order IDs, that would help me debug why you're seeing this

**Comment by rohitramkumar** [comment]
@jonathajones Here are some order IDs in the sandbox env:

4VG15728SX9672816
77V975736F519184N
0P232700C5159452E

**Comment by jonathajones** [comment]
@rohitramkumar  - I found the issue, and working on a resolution. When did you create your rest app?

**Comment by rohitramkumar** [comment]
@jonathajones Cool! 

I'm not exactly sure but I'd say close to a year ago?

**Comment by rohitramkumar** [comment]
@jonathajones any update on this?

**Comment by jonathajones** [comment]
Hi Rohit - No update, but the issue has been raised with the relevant teams. In the meantime, I'll manually add the scope you require to your sandbox account to unblock you.

**Comment by jonathajones** [comment]
You should be able to test the SDK in sandbox now - this won't fix the lack of scopes in prod, but let me know if you need something else in the meantime
